packages/terra-collapsible-menu-view/src/terra-dev-site/doc/example/CollapsibleMenuViewDemo.jsx

```diff
@@ -35,7 +35,13 @@ class CollapsibleMenuViewDemo extends React.Component {
 
   render() {
     return (
-      <CollapsibleMenuView>
+      <CollapsibleMenuView
```

Comment:
I would prefer instead of editing this example, to keep this example as is -- so that it continues to shows the more regular behavior that there is no collapsible-menu-toggle icon button present when all items can visibly fit. 

Instead, add a second example that shows this new feature - and use a smaller number of items, perhaps something like using two buttons, a divider, and a button-group. Something like:

```
  render() {
    return (
      <CollapsibleMenuView
        alwaysCollapsedMenuItems={[
          <CollapsibleMenuView.Item text="Always Collapsed Button 1" key="collapsedButton1" />,
          <CollapsibleMenuView.Item text="Always Collapsed Button 2" key="collapsedButton2" />,
          <CollapsibleMenuView.Item text="Always Collapsed Button 3" key="collapsedButton3" />,
        ]}
      >
        <CollapsibleMenuView.Item text="Collapsible Button 1" key="button1" />
        <CollapsibleMenuView.Item text="Collapsible Button 2" key="button2" />
        <CollapsibleMenuView.Divider key="divider1" />
        <CollapsibleMenuView.ItemGroup key="ViewTypeSelection" selectedKeys={[this.state.displayType]} onChange={this.handleDisplayTypeChange}>
          <CollapsibleMenuView.Item
            icon={<IconTable />}
            text="Table View"
            key="tableView"
            isIconOnly
            shouldCloseOnClick={false}
            isSelected={this.state.displayType === 'tableView'}
          />
          <CollapsibleMenuView.Item
            icon={<IconFlowsheet />}
            text="Expanded View"
            key="expandedView"
            isIconOnly
            shouldCloseOnClick={false}
            isSelected={this.state.displayType === 'expandedView'}
          />
          <CollapsibleMenuView.Item
            icon={<IconVisualization />}
            text="Trending View"
            key="trendingView"
            isIconOnly
            shouldCloseOnClick={false}
            isSelected={this.state.displayType === 'trendingView'}
          />
        </CollapsibleMenuView.ItemGroup>
      </CollapsibleMenuView>
    );
  }
```